scribus/scribusapp.cpp

```diff
@@ -213,7 +217,9 @@ void ScribusQApp::parseCommandLine()
 		std::exit(EXIT_SUCCESS);
 	useGUI = true;
 	//We are going to run something other than command line help
-	for(int i = 1; i < argsc; i++) {
+	QString qtARG_SCRIPTARG_PREFIX = QString(ARG_SCRIPTARG_PREFIX);
+	int i = 1;
+	for( ; i < argsc; i++) { //handle all options (not positional parameters)
```

Comment:
ok. changing "i" into "argi" so it does not look like a convention any more but a real variable with a meaning.